eng/scripts/Language-Settings.ps1

```diff
@@ -210,6 +210,11 @@ function Find-javascript-Artifacts-For-Apireview($artifactDir, $packageName = ""
   [regex]$pattern = "azure-"
   $pkgName = $pattern.replace($packageName, "", 1)
   $packageDir = Join-Path $artifactDir $pkgName "temp"
+  if (-not (Test-Path $packageDir))
```

Comment:
Who are we expecting to create this temp directory?

Reply:
This is created by api-extractor if docmodel is enabled.

Reply:
It might be better to move this error check closer to where we expect it to be created, as if this step fails it isn't clear how someone would go and fix it.

Reply:
This is actually within the api-extractor tool itself. I have added the log with info on how to enable api-extractor json output file. I am going to put this change on hold until I get a confirmation from JS team whether they plan to skip API-extract step for few packages or not.